### ISSUE TITLE
Please provide a LICENSE file

### ISSUE THREAD

**Issue by remicollet** [PHPUnitGoodPractices/polyfill#1]
As MIT license states:

```
The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software.
```

So every project which use the library, without the LICENSE file, is violating the license term.

**Comment by keradus** [comment]
Hi ! Good spot !
I will add it in #2 ;) Thanks!

**Comment by keradus** [comment]
License file was just released under v1.1.0.
